regtests/run_spark_sql.sh

```diff
@@ -52,6 +52,10 @@ export SPARK_LOCAL_HOSTNAME=localhost # avoid VPN messing up driver local IP add
 
 ./setup.sh
 
+if [ -z `which logred` ]; then
+  function logred() { echo $'\u001b'[0\;31m$*$'\u001b'[m; }
+fi
```

Comment:
I'd lean to replace `logred` with `echo`, which is simpler and more portable.

Reply:
Agreed. Switched and force-pushed to streamline git history